src/declarativewebcontainer.cpp

```diff
@@ -266,31 +261,11 @@ void DeclarativeWebContainer::setInputPanelHeight(qreal height)
     }
 }
 
-bool DeclarativeWebContainer::canGoForward() const
```

Comment:
I'd keep this two but read directly from the current page. No need to do caching over here.